libr/bin/format/elf/elf.c

```diff
@@ -2642,295 +2706,7 @@ char *Elf_(r_bin_elf_get_rpath)(ELFOBJ *bin) {
 	return ret;
 }
 
-static bool has_valid_section_header(ELFOBJ *bin, size_t pos) {
```

Comment:
why did you move all this code up? It makes review very hard :( there are way too many changes in this PR and some are just "moving around" others are real changes.

Reply:
the layout was 
-----
rel_cache_new
---
Elf_(r_bin_elf_get_relocs)
---

My only choice was move up or down some function